### PR TITLE
Output min mesh quality to the table

### PR DESCRIPTION
**Pull request by vovannikov**

### CODE REVIEW

File: applications/sintering/include/pf-applications/sintering/postprocessors.h [hunk 1]
```diff
@@ -1925,6 +1925,42 @@ namespace Sintering
...
+                                const DoFHandler<dim> &dof_handler,
+                                const BlockVectorType &solution,
+                                const std::string      output)
+    {
+      Vector<typename BlockVectorType::value_type> quality(
+        dof_handler.get_triangulation().n_active_cells());
+
+      typename BlockVectorType::value_type min_quality = 1.;
+
+      auto callback =
```
Comment (peterrum) [inline applications/sintering/include/pf-applications/sintering/postprocessors.h:1941]:
```suggestion
      const auto callback =
```

File: applications/sintering/include/pf-applications/sintering/postprocessors.h [hunk 2]
```diff
@@ -1925,6 +1925,42 @@ namespace Sintering
...
+      internal::do_estimate_mesh_quality<dim>(dof_handler, solution, callback);
+
+      DataOut<dim> data_out;
+      data_out.attach_triangulation(dof_handler.get_triangulation());
+      data_out.add_data_vector(quality, "quality");
+      data_out.build_patches(mapping);
+      data_out.write_vtu_in_parallel(output, dof_handler.get_communicator());
+
+      min_quality =
+        Utilities::MPI::min(min_quality, dof_handler.get_communicator());
```
Comment (peterrum) [inline applications/sintering/include/pf-applications/sintering/postprocessors.h:1959]: Wouldn't it be easier to take the minimum of `quality` and after that taking the global minimum?

Comment (vovannikov) [reply applications/sintering/include/pf-applications/sintering/postprocessors.h:1959]: It would indeed.

Comment (vovannikov) [reply applications/sintering/include/pf-applications/sintering/postprocessors.h]: @peterrum, have a look, with this more code got reused too.

**Review by peterrum — APPROVED** [review]

**Pull request merged**